tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27,py36
```

Comment:
Since we also support py35, py37 and py38 in travis, please add them here too.

Reply:
This PR is still WIP 🙂